src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabUserManagementService.java

```diff
@@ -246,8 +246,8 @@ public void deleteUser(String login) {
             final var userId = getUserId(login);
             gitlab.getUserApi().deleteUser(userId, true);
         }
-        catch (NullPointerException e) {
-            log.warn("Cannot delete user in GitLab. User does not exist! " + login);
+        catch (GitLabUserDoesNotExistException e) {
+            log.warn("Cannot delete user in GitLab. User does not exist! %s".formatted(login));
```

Comment:
```suggestion
            log.warn("Cannot delete user in GitLab. User does not exist! {}", login);
```
I don't know why it wasn't like that before. (This is the standard logging string format for SLF4J)